packs/opsgenie/README.md

```diff
@@ -0,0 +1,133 @@
+# OpsGenie Pack
+
+To use this intergration pack, you'll need an OpsGenie account and an
```

Comment:
Some two-words description about `opsgenie` would be good, or at least a link.
My thinking is: there are a lot of services and sometimes it's hard to know without googling.

Here is an example with brief description: https://github.com/StackStorm/st2contrib/blob/master/packs/yammer/README.md#yammer-integration-pack
^^ I think that shows good practice.